glam/api/management/commands/import_glean_counts.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+import tempfile
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import connection
+from google.cloud import storage
+
+from glam.api.models import FenixCounts
+
+
+GCS_BUCKET = "glam-fenix-dev"
+MAPPING = {
+    "fenix": {"model": "api.FenixCounts", "apps": ["nightly", "beta", "release"]}
+}
+
+
+def log(message):
+    print(
+        "{stamp} - {message}".format(
+            stamp=datetime.datetime.now().strftime("%x %X"), message=message
+        )
+    )
+
+
+class Command(BaseCommand):
+
+    help = "Imports user counts"
+
+    def handle(self, *args, **options):
+
+        gcs_client = storage.Client()
+        bucket = gcs_client.get_bucket(GCS_BUCKET)
+
+        for product, opts in MAPPING.items():
+            model = apps.get_model(opts["model"])
+
+            for app_id in opts["apps"]:
+                filename = f"glam-extract-org_mozilla_fenix_glam_{app_id}-counts.csv"
```

Comment:
Technically the `app_id` would  be include the org_mozilla_fenix part:

```python
            for channel in opts["apps"]:
                app_id = f"org_mozilla_fenix_glam_{channel}"
                filename = f"glam-extract-{app_id}-counts.csv"
```

This breaks some of the other assumptions in `import_file`.